src/scripts/analyze_missingness.py

```diff
@@ -92,16 +99,30 @@ def count_records_missing_data(
     covid_loincs = pd.read_csv(
         pathlib.Path(__file__).parent / "assets" / "covid_loinc_reference.csv"
     )
+    # Use known COVID vaccine related codes as described in
+    # https://build.fhir.org/ig/HL7/fhir-COVID19Library-ig/branches/master/ValueSet-covid19-cvx-codes-value-set.html
+    # to filter out any non-covid vaccines
+    covid_vax_codes = pd.read_csv(
+        pathlib.Path(__file__).parent / "assets" / "covid_vax_codes.csv", dtype="object"
+    )
 
-    pre_linkage_covid = pre_linkage.merge(covid_loincs, on="loincCode", how="inner")
+    pre_linkage_elr = pre_linkage_elr.merge(covid_loincs, on="loincCode", how="inner")
+    pre_linkage_ecr = pre_linkage_ecr.merge(covid_loincs, on="loincCode", how="inner")
+    pre_linkage_vxu = pre_linkage_vxu.merge(
+        covid_vax_codes, on="vaccineCode", how="inner"
+    )
```

Comment:
How do we solve the case where an ECR contains multiple covid tests and similarly when a VXU contains multiple COVID immunizations?